src/components/Login/components/authError/AuthError.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import styles from './AuthError.css';
+
+export default class AuthError extends Component {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  getErrorMessage(error = {}) {
+    const {
+      code = 'default.error',
+      parameters = [],
+    } = error;
+
+    const values = parameters.reduce((res, { key, value }) => ({ ...res, [key]: value }), {});
+
+    return <FormattedMessage id={`stripes-core.loginServices.login.${code}`} values={values} />;
```

Comment:
Please put each component property to a separate row and wrap this component in a brackets